### PR TITLE
add vulkan-headers/1.2.135.0

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **vulkan-headers/1.2.135.0**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

closes https://github.com/conan-io/conan-center-index/issues/1345

CMake target alias is Vulkan::Headers, but conan doesn't yet provide components. Therefore, I've used a target Vulkan-ValidationLayers can try to find: VulkanHeaders

### CODE REVIEW

**Comment by Croydon** [comment]
Just so that it is not lost, @madebr did quite some work with Vulkan too: https://github.com/bincrafters/community/issues/624

**Comment by SpaceIm** [comment]
I'll take a look. Vulkan SDK of LunarG is a collection of existing libraries and tools (Vulkan-Headers, Vulkan-Loader, Vulkan-ValidationLayers, shaderc, glslang, SPIRV-Headers, etc).

**Comment by Croydon** [comment]
Also https://github.com/bincrafters/community/issues/624#issuecomment-496270884 😄

**Comment by SpaceIm** [comment]
Yes, I think the modular approach is the way to go, so I'll see @madebr work listed in this comment.

File: recipes/vulkan-headers/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy("LICENSE.txt", dst="licenses", src=self._source_subfolder)
+        self.copy("*", dst="include", src=os.path.join(self.source_folder, self._source_subfolder, "include"))
+        self.copy("*", dst=os.path.join("res", "vulkan", "registry"),
+                       src=os.path.join(self.source_folder, self._source_subfolder, "registry"))
+
+    def package_info(self):
+        # CMake Target should be Vulkan::Headers
```
Comment (uilianries) [inline recipes/vulkan-headers/all/conanfile.py:34]:
```suggestion
        # TODO: CMake Target should be Vulkan::Headers
```

In the future we will be able to change it

**Pull request merged**